addons/overpressure/functions/fnc_firedEHOP.sqf

```diff
@@ -21,8 +21,12 @@ TRACE_10("firedEH:",_unit,_weapon,_muzzle,_mode,_ammo,_magazine,_projectile,_veh
 // Retrieve overpressure values
 private _opValues = [_weapon, _ammo, _magazine] call FUNC(getOverPressureValues);
 
-_opValues params ["_dangerZoneAngle", "_dangerZoneRange", "_dangerZoneDamage"];
-TRACE_3("cache",_dangerZoneAngle,_dangerZoneRange,_dangerZoneDamage);
+_opValues params ["_dangerZoneAngle", "_dangerZoneDamage"];
+
+private _dangerZoneRange = (getNumber (_config >> QGVAR(range))) * GVAR(overpressureDistanceCoefficient);
+
+TRACE_3("cache",_dangerZoneAngle,_dangerZoneRange, _dangerZoneDamage);
```

Comment:
Im not sure what this means, is is the space in the macro argument?

ie it should be:
```sqf
TRACE_3("cache", _dangerZoneAngle,_dangerZoneRange,_dangerZoneDamage);
```